dev/com.ibm.ws.microprofile.config.1.3/src/com/ibm/ws/microprofile/config13/interfaces/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+/**
+ * @version 1.0
+ */
+@org.osgi.annotation.versioning.Version("1.0")
```

Comment:
Do you we still need `@version` in the comment when we have the `@Version` annotation? I'm not sure. Also in other package-infos.

Reply:
interesting point ... I *think* that one is used if you generate javadoc and the other is used by bnd to generate the manifests